packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -195,17 +195,46 @@ contract DeploySuperchainOutput is BaseDeployIO {
         else revert("DeploySuperchainOutput: unknown selector");
     }
 
-    // Save the output to a TOML file.
-    // We fetch the output values using external calls to the getters to verify that all outputs are
-    // set correctly before writing them to the file.
-    function writeOutputFile(string memory _outfile) public {
-        string memory key = "dso-outfile";
-        vm.serializeAddress(key, "superchainProxyAdmin", address(this.superchainProxyAdmin()));
-        vm.serializeAddress(key, "superchainConfigImpl", address(this.superchainConfigImpl()));
-        vm.serializeAddress(key, "superchainConfigProxy", address(this.superchainConfigProxy()));
-        vm.serializeAddress(key, "protocolVersionsImpl", address(this.protocolVersionsImpl()));
-        string memory out = vm.serializeAddress(key, "protocolVersionsProxy", address(this.protocolVersionsProxy()));
-        vm.writeToml(out, _outfile);
+    function writeOutputFile(DeploySuperchainInput dsi, string memory _outfile) public {
```

Comment:
taking `dsi` so that inputs can be prepended to outputs.

Reply:
Totally open to changing the structure of this `toml` file. Also adding inline comments might be useful.